_layouts/events.html

```diff
@@ -1,16 +1,51 @@
 ---
 layout: base
 ---
-<div class="events grid-wrapper">
-  <div class="grid__item width-12-12">
-    <h1>Wild<strong>Fly</strong> Conferences</h1>
+<div class="grid-wrapper events">
+  <div class="grid__item width-10-12 width-12-12-m">
+    <h1>Wild<strong>Fly</strong> Events</h1>
     <h2 class="page-subtitle">
-      We host regular conferences for the community with topics related to Wild<strong>Fly</strong>.
+      This page will show the upcoming events of note for Wild<strong>Fly</strong>.
+      You will find here that we are sharing a team calendar and some details related to Wild<strong>Fly</strong> conferences.
     </h2>
+    <h2 id="wildfly-calendar">Wild<strong>Fly</strong> Calendar</h2>
+    <div id="wildfly-org-calendar-container" style="margin: 3em 0"></div>
+    <script type="text/javascript">
+      const options = {
+        src: 'wildflyorg%40gmail.com',
+        ctz: Intl.DateTimeFormat().resolvedOptions().timeZone,
+        showCalendars: '0' // it's only one calendar, so hide the calendar selector
+      };
+      let query = Object.entries(options).map(([key, value]) => `${key}=${value}`).join('&');
+      const url = `https://calendar.google.com/calendar/embed?${query}`;
+      const style = 'display:block;border:none;width:100%;';
+      const attributes = 'width="100%" height="600"';
+      const iframe = `<iframe src="${url}" style="${style}" ${attributes}></iframe>`;
+      document.getElementById('wildfly-org-calendar-container').innerHTML = iframe;
+    </script>
   </div>
 
-  <div class="grid__item width-12-12 sec-title-yellow">
-    <h2>Next Events</h2>
+  <div class="grid__item events-toc">
+    <h3>On this page</h3>
+    <ul>
+      <li><a href="#wildfly-calendar">WildFly Calendar</a></li>
+      <li>
+        <a href="#wildfly-conferences">WildFly Conferences</a>
+        <ul>
+          <li><a href="#next-events">Next Events</a></li>
+          <li><a href="#previous-events">Previous Events</a></li>
```

Comment:
Perhaps for the visible text

Upcoming
Previous

1) "Previous Events" ends up on two lines for me so it doesn't look crisp.
2) In L6 above, the h1 summary block text is "WildFly Events", a broader category, while these TOC items just to the right are about Conferences. So there's cognitive dissonance having the word Event there. Is it the Next Event of all types? No; it's the Next Event of the WildFly Conference type.

Note that I think the existing use of Next Event / Previous Events in the section titles further down in the WildFly Conferences section is fine. By then the reader is past the h1 title and the calendar and in the WildFly Conference section, so there's no confusion.

Reply:
Thanks @bstansberry for the feedback. That makes perfect sense!